qiskit/algorithms/eigensolvers/ssvqe.py

```diff
@@ -0,0 +1,591 @@
+                orthogonal computational basis states.
+            weight_vector (Sequence[float]): A 1D array of real positive-valued numbers to assign
+                as weights to each of the expectation values. If ``None``, then SSVQE will default
+                to [k, k-1, ..., 1].
+            callback (Callable[[int, np.ndarray, Sequence[float], dict[str, Any]], None] | None): A
+                function that can access the intermediate data at each optimization step. These data are
+                the evaluation count, the optimizer parameters for the ansatz, the evaluated mean
+                energies, and the metadata dictionary.
+            check_input_states_orthogonality: A boolean that sets whether or not to check
+                that the value of initial_states passed consists of a mutually orthogonal
+                set of states. If ``True``, then SSVQE will check that these states are mutually
+                orthogonal and return an error if they are not. This is set to ``True`` by default,
+                but setting this to ``False`` may be desirable for larger numbers of qubits to avoid
+                exponentially large computational overhead before the simulation even starts.
+    """
+
+    def __init__(
+        self,
+        estimator: BaseEstimator,
+        k: int | None = 2,
+        ansatz: QuantumCircuit | None = None,
+        optimizer: Optimizer | Minimizer = None,
+        initial_point: Sequence[float] = None,
+        initial_states: list[QuantumCircuit] = None,
+        weight_vector: Sequence[float] | Sequence[int] = None,
+        gradient: BaseEstimatorGradient | None = None,
+        callback: Callable[[int, np.ndarray, Sequence[float], float], None] | None = None,
+        check_input_states_orthogonality: bool = True,
+    ) -> None:
+        """
+        Args:
+            estimator: The estimator primitive.
+            k: The number of eigenstates that the algorithm will attempt to find.
+            ansatz: A parameterized circuit used as Ansatz for the wave function.
+            optimizer: A classical optimizer. Can either be a Qiskit optimizer or a callable
+                that takes an array as input and returns a Qiskit or SciPy optimization result.
+            initial_point: An optional initial point (i.e. initial parameter values)
+                for the optimizer. If ``None`` then VQE will look to the ansatz for a preferred
+                point and if not will simply compute a random one.
+            initial_states: An optional list of mutually orthogonal initial states.
+                If ``None``, then SSVQE will set these to be a list of mutually orthogonal
+                computational basis states.
+            weight_vector: An optional list or array of real positive numbers with length
+                equal to the value of *num_states* to be used in the weighted energy summation
```

Comment:
```suggestion
                equal to the value of ``num_states`` to be used in the weighted energy summation
```
Please check other places.

Reply:
I have now changed this where I have found it.